website/docs/function_utils.md

```diff
@@ -0,0 +1,72 @@
+---
+id: function_utils
+title: Function Utilities
+sidebar_label: Function Utilities
+---
+
+## Function Utility Functions
+
+These functions provide a unified way to retrieve or format a function's parameters
+or get the calling module of a function.
+
+### Functions and Usage
+
+- `build_value_by_func_parameters` this function accepts a `key` parameter in type `str` which contains the parameter's placeholders and then returns out the string by replacing the placeholders with the parameter's actual value.
+
+    #### Usage
+```python
+from core_lib.helpers.func_utils import build_value_by_func_parameters
+
+def foo(param_1, param_2):

```

Comment:
foo again? please let's not use this any more!
suggestion 
`def function_to_format(param_1, param_2):`